lib/src/util/document_event_helper_util.dart

```diff
@@ -0,0 +1,16 @@
+library document_event_helper;
```

Comment:
There are things that were part of ui_core that we didn't bring over, and this is one of them.

It's part of the public API maintain, so we shouldn't add it unless there IS a reason to have it here, IMO.